src/backend.rs

```diff
@@ -186,3 +186,127 @@ mod file {
         }
     }
 }
+
+
+#[cfg(feature = "file-path-backend")]
+pub use self::file_path::FilePath;
+
+#[cfg(feature = "file-path-backend")]
+mod file_path {
+    use std::path::PathBuf;
+
+    use async_trait::async_trait;
+    use tokio::io::{AsyncReadExt, AsyncWriteExt};
+
+    use super::Backend;
+
+    /// A file-path-backed backend. It does not keep the database file open and only access it when needed.
+    /// 
+    /// Saving is first done to a temporary file (with `.tmp` extension appended),
+    /// which is renamed over the real file in case of success.
+    /// 
+    /// You may want to `fsync` the parent directory after saving to ensure the data is actually persisted to disk.
+    /// 
+    /// Note that this implementation is not protected against symlink shenanigans that can redirect the file write elsewhere.
+    ///
+    /// Note: this requires its futures to be executed on the Tokio 0.3 runtime.
```

Comment:
I believe this should be updated to just say "the Tokio runtime", I missed updating the comment on `backend::File` when the update to Tokio 1 was done.

Reply:
Shall this be also changed for `file-backend` in this pull request?

Reply:
If you're happy to fix it there too, please do! It's not a blocker for this PR.

Reply:
Done.